### PR TITLE
Template commands

### PR DESCRIPTION
**Pull request by dwburke**

### CODE REVIEW

File: cmd/cloudTemplateList.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+		if err != nil {
+			lwCliInst.Die(err)
+		}
+
+		zonesResults, err := lwCliInst.LwCliApiClient.Call("bleed/network/zone/list", nil)
+		if err != nil {
+			lwCliInst.Die(err)
+		}
+
+		fmt.Printf("%#v\n", zonesResults)
```
Comment (sgsullivan) [inline cmd/cloudTemplateList.go]: left over debug ?

File: cmd/cloudTemplateList.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+			fmt.Println(", manage-level:", template["manage_level"])
+			fmt.Println("  Zone Availibility:")
+
+			for templateZoneStr, _ := range template["zone_availability"].(map[string]interface{}) {
+				templateZone := cast.ToInt(templateZoneStr)
+				if z, ok := zones[templateZone]; ok {
+					fmt.Printf("    %5d - %s - %s\n", z.Id, z.Name, z.RegionName)
+				}
+
+				//	templatesByZone[templateZone] = append(templatesByZone[templateZone], templateData)
```
Comment (sgsullivan) [inline cmd/cloudTemplateList.go]: old comment?

File: cmd/cloudTemplateList.go (hunk 1 above)
Comment (dwburke) [reply cmd/cloudTemplateList.go]: yeah :blush:

File: cmd/cloudTemplateList.go (hunk 2 above)
Comment (dwburke) [reply cmd/cloudTemplateList.go]: it's gone now :p

**Pull request merged**